CacheService/docker_context/RedisHandler.py

```diff
@@ -0,0 +1,115 @@
+        data, status = await self.client.getRequest(f'{self.dbm_url}/stocks/get/all')
+        if status == 200:
+            for user_id in data:
+                users_data = data[user_id]
+                for dictionary in users_data:
+                    for stock in dictionary:
+                        stock_info = dictionary[stock]
+                        stock_balance = {
+                            "user_id": user_id,
+                            "stock_id": stock,
+                            "stock_amount": stock_info[0],
+                            "stock_reserved": stock_info[1]
+                        }
+                        await self.redis.set("{}_{}".format(user_id, stock), pickle.dumps(stock_balance))
+            return goodResult(msg="Pulled Data", data=''), 200
+        else:
+            return data, status
+
+    async def updateAccountCache(self, user_id):
+        data, status = await self.client.getRequest(f'{self.dbm_url}/funds/get/user/{user_id}')
+        if status == 200:
+            user = {
+                "user_id": user_id,
+                "account_balance": data['available_funds'],
+                "reserved_balance": data['reserved_funds']
+            }
+            await self.redis.set(user_id, pickle.dumps(user))
+            return goodResult(msg="Pulled Data", data=''), 200
+        else:
+            logger.info(f'{__name__} - Error from DBM to get user when updating account cache - {data} ({status})')
+            return data, status
+
+    async def updateStockCache(self, user_id, stock_symbol):
+        data, status = await self.client.getRequest(f'{self.dbm_url}/stocks/get/user/{user_id}?stock_id={stock_symbol}')
+        if not data:
+            err_msg = "The user doesn't hold that stock"
+            logger.info(f'{__name__} - {err_msg}')
+            return errorResult(err_msg, ''), 404
+        if status == 200:
+            for stock in data:
+                stock_info = data[stock]
+                stock_balance = {
+                    "user_id": user_id,
+                    "stock_id": stock,
+                    "stock_amount": stock_info[0],
+                    "stock_reserved": stock_info[1]
+                }
+                await self.redis.set("{}_{}".format(user_id, stock), pickle.dumps(stock_balance))
+            return goodResult(msg="Pulled Data", data=''), 200
+        else:
+            logger.info(f'{__name__} - Error from DBM to getting user stock data - {data} ({status})')
+            return data, status
+
+    async def rSet(self, key, values):
+        await self.redis.set(key, pickle.dumps(values))
+        return
+
+    async def rGet(self, key):
+        result = await self.redis.get(key)
+        result = pickle.loads(result)
+        return result
+
+    async def rExists(self, key):
+        result = await self.redis.exists(key)
+        return result
+
+    async def rDelete(self, key):
+        await self.redis.delete(key)
+        return
```

Comment:
Should this return something?

Reply:
I don't think so since from the Redis level all this can return is the number of rows deleted which we don't end up using. We could ask @MitchPetersen about something for checking this to be more robust?

Reply:
this shouldn't return anything. We're just removing a key,value pair if it exists

We don't ever really need to check to see if something was deleted or not. If it's not deleted, it just won't exist in the first place.